lib/ngx/re.lua

```diff
@@ -193,4 +201,26 @@ function _M.split(subj, regex, opts, ctx, max, res)
 end
 
 
+function _M.opt(option, value)
+    if option == "jit_stack_size" then
+        if not core_regex.regex_cache_is_empty() then
+            return error("Changing jit stack size is not allowed when some " ..
+                         "regexs have already been compiled and cached")
+        end
+
+        local rc = C.ngx_http_lua_ffi_set_jit_stack_size(value)
+
+        if rc == FFI_OK then
+            return
+        elseif rc == FFI_ERROR then
+            return error("PCRE jit stack allocation failed")
+        end
+
+        return error("Received unexpected return code: " .. rc)
+    end
+
+    return error("unrecognized option name")
```

Comment:
A nitpick, but all of the other error messages are capitalized, but they are not in other modules of this library, nor are they in ngx_lua iirc.

Better to stay consistent in our API, and maybe un-capitalize the other errors thrown by this function?